src/Identity/EntityFrameworkCore/src/IdentityDbContext.cs

```diff
@@ -41,6 +41,26 @@ public class IdentityDbContext<TUser> : IdentityDbContext<TUser, IdentityRole, s
         protected IdentityDbContext() { }
     }
 
+    /// <summary>
+    /// Base class for the Entity Framework database context used for identity.
+    /// </summary>
+    /// <typeparam name="TUser">The type of the user objects.</typeparam>
+    /// <typeparam name="TKey">The type of the key for the user object.</typeparam>
+    public class IdentityDbContext<TUser, TKey> : IdentityDbContext<TUser, IdentityRole<TKey>, TKey> where TUser : IdentityUser<TKey>
+        where TKey : IEquatable<TKey>
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="IdentityDbContext"/>.
+        /// </summary>
+        /// <param name="options">The options to be used by a <see cref="DbContext"/>.</param>
+        public IdentityDbContext(DbContextOptions options) : base(options) { }
```

Comment:
This looks incorrect to me. A public constructor should use `DbContextOptions<TContext>`. If this class is intended to also be a base type, then it should get a protected constructor that takes `DbContextOptions`.

Reply:
@ajcvickers - I just followed the pattern of all of the other existing contexts in the file.  Let me know if you still think this should be changed.

Reply:
@StevenRasmussen Any context that may be both used directly and inherited from should have both constructors. If there are other context classes that have this requirement but do not follow this pattern, then they should be changed as well.